src/pages/Account/accountUtils.ts

```diff
@@ -18,6 +19,15 @@ const expressions = {
   type: new RegExp('^(?<type>[A-Za-z]+)'),
 }
 
+export const bridgeIdToDisplayNameMap: { [k in BridgeList]: string } = {
```

Comment:
Also, we usually type global constant names with UPPER_CASE convention

Reply:
Agree here